src/megachatapi_impl.cpp

```diff
@@ -692,6 +692,62 @@ void MegaChatApiImpl::sendPendingRequests()
             });
             break;
         }
+        case MegaChatRequest::TYPE_PREVIEW_CHAT_LINK:
+        {
+            const char *link = request->getText();
+            string parsedLink = link;
+
+            size_t pos = parsedLink.find("#chat_");
+            if (pos == string::npos)
+            {
+                errorCode = MegaChatError::ERROR_ARGS;
+                break;
+            }
+
```

Comment:
Despite it's not a problem to have those numbers hardcoded here, it's good to have a comment to guide the reader. Please add a line with the expected format of the link, something like:
`#chat_<public-handle><chat-key><\0>`

Reply:
Done!